testdata/fuzz/FuzzNGAP/ci0000

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
```

Comment:
Hi @ShouheiNishi 

I'm wondering how did you generate all the `ciXXXX` files.

Thank you.